app/atoms.html

```diff
@@ -785,6 +785,9 @@ <h3>Badges</h3>
                                 <h4 class="example-name">Single badge</h4>
                                 <div class="component">
                                     <span class="badge">new</span>
+                                    <span class="badge badge-active">active</span>
```

Comment:
what are these badges?

Reply:
These are badges that were in the css but had not been put in the style guide. Figured it would be helpful for use in future designs.